src/transformations.jl

```diff
@@ -121,21 +121,20 @@ struct UnitRangeTransform{T<:Real}  <: DataTransform
 end
 
 function Base.getproperty(t::UnitRangeTransform, p::Symbol)
-    if p == :indim || p == :outdim
+    if p === :indim || p === :outdim
         return t.dim
     else
         return getfield(t, p)
     end
 end
 
 # fit a unit transform
-function fit(::Type{UnitRangeTransform}, X::AbstractArray{T,2}; unit::Bool=true) where T<:Real
+function fit(::Type{UnitRangeTransform}, X::AbstractArray{T,2}; unit::Bool=true) where {T<:Real}
     d, n = size(X)
 
     tmin = zeros(T, d)
-    tmax = zeros(T, d)
-    copyto!(tmin, X[:, 1])
-    copyto!(tmax, X[:, 1])
+    copyto!(tmin, @view X[:, 1])
```

Comment:
As I noted, wouldn't `tmin = X[:, 1]` be enough?

Reply:
Right, copy.